tests/tests-asn1c-compiler/07-int-OK.asn1

```diff
@@ -13,7 +13,9 @@ BEGIN
 
 	int123456 INTEGER ::= 123456
 
-	Interval ::= INTEGER (1..int123456)
```

Comment:
with the files that originally came with the project, I never changed the ASN.
If I wanted to change the ASN1, I always made a new file e.g. 07a...
This was incase it was ever upstreamed, I would not be breaking someone else's test

Reply:
done